### PR TITLE
Create migrating-to-v2.md

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Per [this internal discussion](https://dbt-labs.slack.com/archives/C04LEEWFG3F/p1676560775779259), we need a migration notes for migrating from v4 to v2.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Migrating to dbt Cloud Administrative API v2"
+description: "You should migrate to API v2 while we deprecate API v4 "
+sidebar_label: "Migrating to API v2"
+id: "admin-cloud-api"
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
id: "migrating-to-v2"
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should account when querying your data. 
+
+## Key differences
+
+When using the `List runs` endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Migrating to dbt Cloud Administrative API v2"
+description: "You should migrate to API v2 while we deprecate API v4 "
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should account when querying your data.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your data. 
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: "Migrating to dbt Cloud Administrative API v2"
+description: "You should migrate to API v2 while we deprecate API v4 "
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your data.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=["trigger"]&order_by=-id
+```
+
+Differences in responses include:
+
+| Property | API v4    | API v2     |
+|---------|-----------|-------------|
+| Use sort by `-id` to reverse the sort order | Defaults to order by most recent | Defaults to order by least recent |
+| timestamps | Unix timestamps | ISO strings |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Timestamps | Unix timestamps | ISO strings |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=["trigger"]&order_by=-id
+```
+
+Differences in responses include:
+
+| Property | API v4    | API v2     |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Property description | API v4    | API v2     |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=["trigger"]&order_by=-id
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------|-----------|-------------|
+| Use sort by `-id` to reverse the sort order | Defaults to order by most recent | Defaults to order by least recent |
+| Timestamps | Unix timestamps | ISO strings |
+| `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
+| Status | `status` property |  Maps to `status_humanized` |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Returns runs with the specified status | `status` property |  Maps to `status_humanized` |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------|-----------|-------------|
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
|---------------------|-----------|-------------|
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+| Use sort by `-id` to reverse the sort order | Defaults to order by most recent | Defaults to order by least recent |
+| Timestamps | Unix timestamps | ISO strings |
+| `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
+| Status | `status` property |  Maps to `status_humanized` |
+| Replace | `replace` property | Maps to the `trigger` property |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Related field included with run | `replace` property | Maps to the `trigger` property |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------|-----------|-------------|
+| Use sort by `-id` to reverse the sort order | Defaults to order by most recent | Defaults to order by least recent |
+| Timestamps | Unix timestamps | ISO strings |
+| `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Updated IDs: `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------------------|-----------|-------------|
+| Use sort by `-id` to reverse the sort order | Defaults to order by most recent | Defaults to order by least recent |
+| Timestamps | Unix timestamps | ISO strings |
+| Updated IDs: `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
+| Returns runs with the specified status | `status` property |  Maps to `status_humanized` |
+| Related field included with run | `replace` property | Maps to the `trigger` property |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Include related field with run | `replace` property | Maps to the `trigger` property |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------------------|-----------|-------------|
+| Use sort by `-id` to reverse the sort order | Defaults to order by most recent | Defaults to order by least recent |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Reverse sort order when you use sort by `-id`  | Defaults to order by most recent | Defaults to order by least recent |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+| Timestamps | Unix timestamps | ISO strings |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Change timestamps | Unix timestamps | ISO strings |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+| Updated IDs: `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Update to IDs: `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+| Returns runs with the specified status | `status` property |  Maps to `status_humanized` |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Return runs with the specified status | `status` property |  Maps to `status_humanized` |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------------------|-----------|-------------|
+| Reverse sort order when you use sort by `-id`  | Defaults to order by most recent | Defaults to order by least recent |
+| Change timestamps | Unix timestamps | ISO strings |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| Update to timestamps | Unix timestamps | ISO strings |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+```
+
+Differences in responses include:
+
+| Property description | API v4    | API v2     |
+|---------------------|-----------|-------------|
+| Reverse sort order when you use sort by `-id`  | Defaults to order by most recent | Defaults to order by least recent |
+| Update to timestamps | Unix timestamps | ISO strings |
+| Update to IDs: `id`, `environment_id`, `account_id`, `project_id`, `job_id` | Values are the same, but they are strings | Values are the same, but they are numeric |
+| Return runs with the specified status | `status` property |  Maps to `status_humanized` |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| New property for returning runs with the specified status | `status` property |  Maps to `status_humanized` |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+| Include related field with run | `replace` property | Maps to the `trigger` property |
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
| New property for including related field with run | `replace` property | Maps to the `trigger` property |
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 20]
```diff
@@ -0,0 +1,27 @@
...
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
```
Comment (iktl) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]: One minor nit, do we have precedent yet for handling URLs where we need to reference emea/au/us separately?

Comment (mirnawong1) [reply website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]: i _think_ Leona (who's OOO) has created this placeholder text that would follow the example url:

> Replace `https://cloud.getdbt.com` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint is `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`


another example [here](https://docs.getdbt.com/docs/dbt-cloud-apis/metadata-api#browse-the-api)

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 21]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `cloud.getdbt.com` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`:
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 22]
```diff
@@ -0,0 +1,27 @@
...
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `cloud.getdbt.com` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`:
+
+```shell
+GET https://cloud.getdbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
GET https://{YOUR_ACCESS_URL}/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 23]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `cloud.getdbt.com` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `{YOUR_ACCESS_URL}` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`:
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 24]
```diff
@@ -0,0 +1,27 @@
...
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `{YOUR_ACCESS_URL}` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`:
+
+```shell
+GET https://{YOUR_ACCESS_URL}/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md:23]: For example, if your region is EMEA multi-tenant and your account ID is `001`, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/001/runs/?include_related=[%22trigger%22]&order_by=-id`

Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md:23]:
```suggestion

 For example, if your region is EMEA multi-tenant and your account ID is `001`, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/001/runs/?include_related=[%22trigger%22]&order_by=-id`
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 25]
```diff
@@ -0,0 +1,27 @@
...
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
+
+When migrating from dbt Cloud Administrative API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account. 
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `{YOUR_ACCESS_URL}` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan. For example, if your region is EMEA multi-tenant, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id`:
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `{YOUR_ACCESS_URL}` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan.
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 26]
```diff
@@ -0,0 +1,28 @@
...
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `{YOUR_ACCESS_URL}` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan.
+
+```shell
+GET https://{YOUR_ACCESS_URL}/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
+ For example, if your region is EMEA multi-tenant and your account ID is `001`, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/001/runs/?include_related=[%22trigger%22]&order_by=-id`
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
For example, if your region is EMEA multi-tenant and your account ID is `001`, your endpoint would be  `GET https://emea.dbt.com/api/v2/accounts/001/runs/?include_related=[%22trigger%22]&order_by=-id`
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md (hunk 20 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]: I've addressed this! Thanks for bringing it up.

File: website/docs/docs/dbt-cloud-apis/admin-cloud-api.md [hunk 27]
```diff
@@ -10,4 +10,4 @@ The dbt Cloud Administrative API is enabled by default for _Team_ and _Enterpris
 - Manage your dbt Cloud account
 - and more
 
-Reference documentation for the dbt Cloud Administrative v2 API can be found [here](/dbt-cloud/api-v2).
+You can use [dbt Cloud Administrative API v2 reference documentation](/dbt-cloud/api-v2) to help you access the API. For information about migrating from API v4 to API v2, refer to [Migrating to dbt Cloud Administrative API v2](/docs/dbt-cloud-apis/migrate-to-v2).
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/admin-cloud-api.md]:
```suggestion
You can use [dbt Cloud Administrative API v2 reference documentation](/dbt-cloud/api-v2) to help you access the API. For information about migrating from API v4 to API v2, refer to [Migrating to dbt Cloud Administrative API v2](/docs/dbt-cloud-apis/migrating-to-v2).
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 28]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Migrating to dbt Cloud Administrative API v2"
+description: "You should migrate to API v2 while we deprecate API v4 "
+sidebar_label: "Migrating to API v2"
+id: "migrating-to-v2"
+---
+
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md:7]: Should we include a blurb here about why we're deprecating V4 and asking them to step down to V2 (maybe even why there is no V3)? Security teams might see reverting to a lower version as a red flag.

File: _redirects [hunk 29]
```diff
@@ -15,7 +15,7 @@
 /docs/dbt-cloud/cloud-ide/viewing-docs-in-the-ide /docs/get-started/develop-in-the-cloud 301
 /docs/dbt-cloud/cloud-overview /docs/get-started/getting-started/set-up-dbt-cloud 301
 /docs/dbt-cloud/using-dbt-cloud/artifacts /docs/deploy/artifacts 301
-/dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 301
+# /dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 301!  commenting out until we deprecate v4. We need the bang only if v4 still exists.
```
Comment (runleonarun) [inline _redirects]:
```suggestion
## /dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 301!  commenting out until we deprecate v4. We need the bang only if v4 still exists.
```

File: _redirects [hunk 30]
```diff
@@ -15,7 +15,7 @@
 /docs/dbt-cloud/cloud-ide/viewing-docs-in-the-ide /docs/get-started/develop-in-the-cloud 301
 /docs/dbt-cloud/cloud-overview /docs/get-started/getting-started/set-up-dbt-cloud 301
 /docs/dbt-cloud/using-dbt-cloud/artifacts /docs/deploy/artifacts 301
-/dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 301
+## /dbt-cloud/api-v4 /docs/dbt-cloud-apis/admin-cloud-api 301!  commenting out until we deprecate v4. We need the bang only if v4 still exists.
```
Comment (runleonarun) [inline _redirects]:
```suggestion
```

File: website/docs/docs/dbt-cloud-apis/migrating-to-v2.md [hunk 31]
```diff
@@ -0,0 +1,33 @@
...
+In an attempt to provide an improved dbt Cloud Administrative API experience, the dbt Cloud API v4 will be deprecated by April 30th, 2023. We suggest you migrate to dbt Cloud Administrative API v2. When migrating from API v4 to API v2, there are a few differences you should consider when querying your dbt Cloud account.
+
+## Key differences
+
+When using the [List runs](/dbt-cloud/api-v2#tag/Runs) endpoint, you can include triggered runs and sort by ID. You can use the following request in v2 to get a similar response as v4, replacing the `{accountId}` with your own and `{YOUR_ACCESS_URL}` with the appropriate [Access URL](https://docs.getdbt.com/docs/deploy/regions-ip-addresses) for your region and plan:
+
+```shell
+GET https://{YOUR_ACCESS_URL}/api/v2/accounts/{accountId}/runs/?include_related=[%22trigger%22]&order_by=-id
+```
+
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud-apis/migrating-to-v2.md]:
```suggestion
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**